frb_codegen/src/config.rs

```diff
@@ -17,10 +17,11 @@ use toml::Value;
 pub struct RawOpts {
     /// Path of input Rust code
     #[structopt(short, long)]
-    pub rust_input: String,
+    rust_input: Vec<String>,
+
```

Comment:
Unneeded newline which is inconsistent?

Reply:
fix it in 9e38c2995f796d3e5c35b6bbaa631d62645325dc.